src/sagas/task.saga.js

```diff
@@ -332,6 +337,12 @@ export const getDevServerCommand = (
   }
 };
 
+export const stripUnusableControlCharacters = (text: string) =>
+  // The control character '[1G' is meant to "Clear vertical tab stop at
+  // current line". Unfortunately, it isn't correctly parsed, and shows
+  // up in the output as "G".
+  text.replace(/\[1G/g, '');
```

Comment:
I think this feels reasonable to me... of course the downside is that if the output genuinely contains the string "[1G", it'll be stripped out... but that feels unlikely.

I tried making it a bit more specific, but ran into problems:
- I tried making it `/^\[1G/g`, so it would only strip out prefixes, but it wasn't always the first control character.
- It's generally paired with another control character, `[2F`, but they have some invisible character between them, so I couldn't just search for that string and then remove its sibling.

If we DO get some complains about confusing output missing the string `[1G`, I think we can spend more time digging, but my feeling is that we'll never run into that situation.

Reply:
This feels safe to me. Thankfully, control characters have very esoteric syntax - whether this was on purpose to avoid accidental use or not, it's very convenient here.